src/main/java/org/alliancegenome/curation_api/model/entities/Allele.java

```diff
@@ -58,7 +58,6 @@
 @ToString(exclude = { "alleleGeneAssociations", "alleleDiseaseAnnotations", "alleleMutationTypes", "alleleSymbol", "alleleFullName", "alleleSynonyms", "alleleSecondaryIds", "alleleInheritanceModes", "alleleFunctionalImpacts", "alleleGermlineTransmissionStatus", "alleleDatabaseStatus", "alleleNomenclatureEvents" }, callSuper = true)
 @AGRCurationSchemaVersion(min = "1.7.3", max = LinkMLSchemaConstants.LATEST_RELEASE, dependencies = { GenomicEntity.class }, partial = true)
 @Table(indexes = { @Index(name = "allele_inCollection_index", columnList = "inCollection_id"), })
-@JsonIdentityInfo(generator = ObjectIdGenerators.PropertyGenerator.class, property = "curie")
```

Comment:
This annotation shouldn't have been kept in and is causing the error on the disease annotation table at the moment.